script.js

```diff
@@ -2,40 +2,23 @@ const btn = document.querySelector('#loginForm');
 
 const CONFIG = {
   API_URL: '/api/v1/submit',
-  TOKEN_KEY: '',
 };
 
-btn.addEventListener('submit', async (e) => {
-  e.preventDefault();
-
-  /* btn.classList.add("spinner"); */
-  document.getElementById('spinner').style.display = 'block';
-  const username = document.getElementById('username').value;
-  const password = document.getElementById('password').value;
-  const errorDisplay = document.getElementById('errorMessage');
-  errorDisplay.style.display = 'none';
-
-  const result = await handleLogin(username, password);
-
-  try {
-    if (result.status) {
-      // Post to the API_URL
-      window.location.href = '/dashboard';
-    } else {
-      errorDisplay.textContent = result.message;
-      errorDisplay.style.display = 'block';
+const handleApiError = (error) => {
+  if (error.status) {
```

Comment:
This line will always return true, as it does not perform an equality operation. I believe you meant error.status==='fail'.